code/modules/client/preferences_savefile.dm

```diff
@@ -47,20 +47,6 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		outline_color = COLOR_BLUE_GRAY
 	if(current_version < 31)
 		auto_fit_viewport = TRUE
-	if(current_version < 32)
```

Comment:
you might want to remove currently saved gear in savefiles.
Files are treated as big lists so just remove the index.